packages/core/addon/components/pick-container.js

```diff
@@ -13,6 +21,8 @@
 import Ember from 'ember';
 import Layout from '../templates/components/pick-container';
 
+const { get, computed } = Ember;
```

Comment:
`import { computed, get } from '@ember/object';`

Reply:
Odd, i don't even remember adding this import o_O